misc/python/materialize/mzcompose/services/schema_registry.py

```diff
@@ -36,37 +37,37 @@ def __init__(
             f"SCHEMA_REGISTRY_KAFKASTORE_BOOTSTRAP_SERVERS={bootstrap_servers}",
             *environment_extra,
         ]
+        config: ServiceConfig = {
+            "image": f"{image}:{tag}",
+            "ports": [port],
+            "networks": {"default": {"aliases": aliases}},
+            "environment": environment,
+            "depends_on": {
+                **{host: {"condition": "service_healthy"} for host, _ in kafka_servers},
+                **{s: {"condition": "service_started"} for s in depends_on_extra},
+            },
+            "healthcheck": {
+                "test": [
+                    "CMD",
+                    "curl",
+                    # We provide credentials in case the schema registry is
+                    # configured to require HTTP authentication, as there's
+                    # no health check endpoint that's excluded from
+                    # authentication requirements. The credentials are
+                    # safely ignored if the schema registry is not
+                    # configured to require them.
+                    "-fu",
+                    "materialize:sekurity",
```

Comment:
This `sekurity` string may now be present at several places in the codebase. Consider replacing with a named constant.

Reply:
Ops you just moved that code, nevermind.